tests/eventdispatcher/PatientFilterEventHookTest/acl/acl_setup.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require_once($GLOBALS["webserver_root"].'/interface/globals.php');
+require_once($GLOBALS["webserver_root"].'/library/acl_upgrade_fx.php');
+
+//Ensure that phpGACL has been installed
+require_once($GLOBALS["webserver_root"].'/library/acl.inc');
+
+if (isset($phpgacl_location)) {
+    include_once("$phpgacl_location/gacl_api.class.php");
+    $gacl = new gacl_api();
+} else {
+    die("You must first set up library/acl.inc to use phpGACL!");
+}
```

Comment:
just do following for above block:
```
require_once($GLOBALS["webserver_root"].'/interface/globals.php');
require_once($GLOBALS["webserver_root"].'/library/acl_upgrade_fx.php');
require_once($GLOBALS["webserver_root"].'/library/acl.inc');
```
(that is all that should be needed)

Reply:
Note the above line for globals.php likely will not work since $GLOBALS not yet defined. Should use absolute path there as done in rest of codebase.

Reply:
Without using the file "gacl_api.class.php", the program throws an error.

Reply:
> Note the above line for globals.php likely will not work since $GLOBALS not yet defined. Should use absolute path there as done in rest of codebase.

At a given point in the code, this variable is already initialized